server/src/microservices/processors/metadata-extraction.processor.ts

```diff
@@ -336,6 +336,19 @@ export class MetadataExtractionProcessor {
         await this.extractEmbeddedVideo(asset, offset, null, fileCreatedAt);
       }
     }
+
+    const exifProjectionType = getExifProperty('ProjectionType');
+    let projectionType: ProjectionType = ProjectionType.NONE;
+
+    if (exifProjectionType) {
+      const exifPropertyUpper = exifProjectionType.toUpperCase();
+
+      if (exifPropertyUpper in ProjectionType) {
+        projectionType = exifPropertyUpper as ProjectionType;
+      }
+    }
+
+    newExif.projectionType = projectionType;
```

Comment:
```suggestion
    const projectionType = getExifProperty('ProjectionType');
    if (projectionType) {
      newExif.projectionType = String(projectionType).toUpperCase();
    }
```

Reply:
newExif.projectionType is a enum. What will happen in case `String(projectionType).toUpperCase();` does not exist in this enum?

Reply:
The database type is a varchar, right? The entity type should be `ProjectionType | string`.

Reply:
If we only accept valid values we should use pg enum. If we accept other values we should use a varchar and store the raw value (maybe transformed to uppercase). IMO the projection type is just a hint of what valid types are available to be used, but the value is just a string. I don't know if it is possible/easy to use enum in the web and flutter without getting an error for other values.

Reply:
I think everything would be quite a bit easier if you just stored it as a string everywhere in the server codebase, and had the property in the response dto be a string.

You can define the enum in the web to use for your if statements. Happy to discuss more in discord if you want to chat there instead of back and forth here.

Reply:
I would suggest to keep enum | str.

As far as I understood, there are just 2 most popular projection types:equirectangular and cylinder